source/_integrations/telegram.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to add Telegram notifications to Home Assistant
 ha_category:
   - Notifications
 ha_release: 0.7.5
+ha_iot_class: Cloud Polling
```

Comment:
How did you get to this conclusion?

Reply:
The telegram notification platform requires the telegram_bot integration to work with.
https://www.home-assistant.io/integrations/telegram_bot/#telegram-notification-platform

The example configuration for the Telegram_bot integration described in the Telegram (notify) integration uses the polling method.
https://www.home-assistant.io/integrations/telegram#configuration

You can also configure the Telegram_bot integration to use the webhook method.
https://www.home-assistant.io/integrations/telegram_bot/

Reply:
Right, that makes it polling. Thanks for clarifying! 👍